### PR TITLE
Final Fix Iframe problem

### PR DESCRIPTION
**Pull request by pamungkaski**
Fancy Things
screen shot 2017-08-22 at 2 35 10 pm
screen shot 2017-08-22 at 3 00 17 pm
@jayvdb

### CODE REVIEW

**Comment by jayvdb** [comment]
I havent tested, but I am pretty sure that this will not work well, as each mobile device can render the input controls using any height they want, adding fancy mechanisms for helping the user select the value.  It needs to be computed.

**Comment by pamungkaski** [comment]
@jayvdb Fancy things? Done

File: assets/js/b24-form-interactions.js [hunk 1]
```diff
@@ -13,9 +25,18 @@ var onFormLoadInteractions = {
       formElement.style.display = "none";
       bdy.style.overflow = "scroll";
     }
-    var b24Form = document.getElementById('bx_form_iframe_8');
     b24Form.scrolling = 'yes';
-    b24Form.style.height = '400vh';
+    b24Form.style['overflow-y'] = 'scroll';
+    formElement.style.height = (windowHeight) + 'px';
+    console.log(b24Form.style.height);
+    if(b24Form.style.height === '200px') {
```
Comment (jayvdb) [inline assets/js/b24-form-interactions.js]: Could you a comments explaining these magic numbers 200, 300 and 600.
Also store any literal values in a constant variable that is wisely named.

File: index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
     <link href="assets/css/material-kit.css" rel="stylesheet" />
   </head>
   <body class="landing-page" style="overflow: scroll">
-    <div id="form_bit24"style="display: none; position: absolute; width: 100%; min-height: 100%; background-color: rgba(0,0,0,0.5); overflow-y: scroll;  z-index: 1000000; top: 0; right: 0; bottom: 0; left: 0;padding: 20px">
+    <div id="form_bit24"style="display: none; position: absolute; background-color: rgba(0,0,0,0.5); overflow-y: scroll;  z-index: 1000000; top: 0; right: 0; bottom: 0; left: 0;padding: 20px">
```
Comment (jayvdb) [inline index.html]: Add a space after `24"`

Also, would be better to use css for the style.

**Comment by jayvdb** [comment]
I've very quickly played with this, and it doesnt seem to work properly with Chrome or Firefox on a phone.

**Pull request merged**